From 7480f6fc939fafe83163cbc228fab02bf343d6e9 Mon Sep 17 00:00:00 2001
From: Yusuke Goda <yusuke.goda.sx@renesas.com>
Date: Tue, 21 Aug 2018 12:10:10 +0900
Subject: [PATCH 687/909] arm64: dts: r8a7795-h3ulcb-4x2g: Overwrite dma-ranges
 of pcie nodes for R-Car H3 ES2.0/3.0

commit 6092e813c0be65afb59776e026fb79ae6b6e806e from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch overwrites dma-ranges property of pcie nodes
due to memory map change in R-Car H3 ES2.0/3.0 (2 x 2 GiB)

Signed-off-by: Yusuke Goda <yusuke.goda.sx@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-h3ulcb-4x2g.dts | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb-4x2g.dts b/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb-4x2g.dts
index 29ef263f7120..15c034ba2b7a 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb-4x2g.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb-4x2g.dts
@@ -35,3 +35,13 @@
 		reg = <0x7 0x00000000 0x0 0x80000000>;
 	};
 };
+
+&pciec0 {
+	/* Map all possible DDR as inbound ranges */
+	dma-ranges = <0x42000000 0 0x40000000 0 0x40000000 0 0x80000000>;
+};
+
+&pciec1 {
+	/* Map all possible DDR as inbound ranges */
+	dma-ranges = <0x42000000 0 0x40000000 0 0x40000000 0 0x80000000>;
+};
-- 
2.17.1

